tests/test_fabric_port.py

```diff
@@ -0,0 +1,50 @@
+from swsscommon import swsscommon
+from dvslib.dvs_database import DVSDatabase
+
+
+class TestVirtualChassis(object):
+    def test_voq_switch_fabric_link(self, vst):
+        """Test fabric link manual isolation commands in VOQ switch.
+
+        By issuing config fabric port isolation command, the value
+        of isolateStatus field in config_db get changed. This test validates appl_db
+        updates of a fabric link isolateStatus as the value in config_db changed.
+        """
+
+        dvss = vst.dvss
+        for name in dvss.keys():
+            dvs = dvss[name]
+            # Get the config info
+            config_db = dvs.get_config_db()
+            metatbl = config_db.get_entry("DEVICE_METADATA", "localhost")
+
+            cfg_switch_type = metatbl.get("switch_type")
+            if cfg_switch_type == "fabric":
```

Comment:
do we need to test if `switch_type == VOQ` ?

Reply:
> do we need to test if `switch_type == VOQ` ?

In the current vs test environment, the fabric related setup ( ports, counters and etc. ) are all under the cfg_switch_type fabric. So I am having this if check here. 

If you really want to start testing the VOQ, that will need some more infra changes, if so, I can do in a separate PR later to cover them there if you are ok.  thank you